<launch>

	<!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3_lite"/>
    <arg name="start_delay_seconds" default="0"/>

  	<node name="camera_server" pkg="kortex_tools" type="camera_server.py" output="screen" launch-prefix="bash -c 'sleep $(arg start_delay_seconds); $0 $@' ">
  	</node>

</launch>
